packages/components/src/components/hive-navigation/index.tsx

```diff
@@ -127,15 +133,26 @@ export function HiveNavigation({
               <CompanyMenu>{companyMenuChildren}</CompanyMenu>
             </NavigationMenuContent>
           </NavigationMenuItem>
-          <NavigationMenuItem className="flex">
-            <NavigationMenuLink
-              href={isHive ? '#pricing' : 'https://the-guild.dev/graphql/hive#pricing'}
-              className="font-medium"
-            >
-              Pricing
-            </NavigationMenuLink>
-          </NavigationMenuItem>
+          {navLinks ? (
+            navLinks.map(({ href, children }, i) => (
+              <NavigationMenuItem key={i} className="flex">
+                <NavigationMenuLink href={href} className="font-medium">
+                  {children}
+                </NavigationMenuLink>
+              </NavigationMenuItem>
+            ))
+          ) : (
+            <NavigationMenuItem className="flex">
+              <NavigationMenuLink
+                href={isHive ? '#pricing' : 'https://the-guild.dev/graphql/hive#pricing'}
+                className="font-medium"
+              >
+                Pricing
+              </NavigationMenuLink>
+            </NavigationMenuItem>
+          )}
         </NavigationMenuList>
```

Comment:
```suggestion
navLinks ||= [
  {
    href: isHive ? '#pricing' : 'https://the-guild.dev/graphql/hive#pricing',
    children: 'Pricing'
  }
]
```

similar as you did for `logo ||=`